content/en/sheets/_index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Sheets
+further_reading:
+- link: "/sheets/functions_operators"
+  tag: "Documentation"
+  text: "Functions and Operators"
+---
+
+{{< callout url="https://www.datadoghq.com/private-beta/datadog-sheets/" btn_hidden="false" header="Join the Beta!">}}
+Sheets is available in <strong>private beta</strong>. To qualify for this product beta, you should have existing use cases where you currently use spreadsheets (such as Excel or Google Sheets) with Datadog’s metrics, logs, or RUM data. If you're interested in this feature, complete the form to request access.
+{{< /callout >}}
+
+## Overview
+
+Sheets is a spreadsheet tool that connects to Datadog data, enabling you to perform complex data analysis and build reports without requiring technical expertise. It allows teams to use familiar spreadsheet functions like lookups, pivot tables, and calculations on Datadog data, so you don’t have to export and use another tool with stale data. 
```

Comment:
```suggestion
Sheets is a spreadsheet tool that you can populate with Datadog data, enabling you to perform complex analysis and build reports without requiring technical expertise. It allows teams to use familiar spreadsheet functions like lookups, pivot tables, and calculations on Datadog data, so you don’t have to export and use another tool with stale data. 
```
"connects to" made me think it was more of a Google Sheets integration or similar, so I tried to reword to make the tool's purpose clearer. It's a tough one to write, though!